src/constants/logos.ts

```diff
@@ -0,0 +1,10 @@
+export const LOGOS: { [key: string]: string } = {
+  python:
```

Comment:
```suggestion
  PYTHON:
```
이것도 위와 같이 변경해야 할까요? 서버랑 맞추어야 하겠죠?

Reply:
상수니까 컨벤션 맞춰두겠습니다!